ml/src/test/scala/ai/h2o/sparkling/ml/algos/H2OStackedEnsembleTestSuite.scala

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.ml.algos
+
+import ai.h2o.sparkling.ml.internals.H2OModel
+import ai.h2o.sparkling.{SharedH2OTestContext, TestUtils}
+import org.apache.spark.ml.{Pipeline, PipelineModel}
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.col
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{FunSuite, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class H2OStackedEnsembleTestSuite extends FunSuite with Matchers with SharedH2OTestContext {
+
+  override def createSparkSession(): SparkSession = sparkSession("local[*]")
+
+  private lazy val dataset = spark.read
+    .option("header", "true")
+    .option("inferSchema", "true")
+    .csv(TestUtils.locate("smalldata/prostate/prostate.csv"))
+    .withColumn("CAPSULE", col("CAPSULE").cast("string"))
+
+  private lazy val trainingDataset = dataset.limit(300).cache()
+  private lazy val testingDataset = dataset.except(trainingDataset).cache()
+
+  private val foldsNo = 5
+
+  test("Create Stacked Ensemble model using cross validations") {
+
+    val glm = getGlm()
+    val gbm = getGbm()
+
+    val ensemble = new H2OStackedEnsemble()
+      .setBaseAlgorithms(Array(glm, gbm))
+      .setMetalearnerAlgorithm("GLM")
+      .setLabelCol("CAPSULE")
+      .setSeed(42)
+      .setScoreTrainingSamples(0)
+
+    val ensembleModel = ensemble.fit(trainingDataset)
+
+    ensembleModel.getMetalearnerAlgorithm() shouldBe "glm"
+
+    val predictions = ensembleModel.transform(testingDataset)
+    predictions.distinct().count() shouldBe 80
```

Comment:
Shouldn't we check distinct count only on the detailed response column? Features might be different but we must make sure that the model doesn't give the same prediction for all rows